src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/ResetAzureVirtualNetworkGatewayConnectionSharedKeyCommand.cs

```diff
@@ -44,7 +44,7 @@ public class ResetAzureVirtualNetworkGatewayConnectionSharedKeyCommand : Virtual
          ValueFromPipelineByPropertyName = true,
          HelpMessage = "The virtual network connection reset shared key length")]
         [ValidateNotNullOrEmpty]
-        public uint KeyLength { get; set; }
+        public int KeyLength { get; set; }
```

Comment:
As we talked offline, please revert this change and update the PR

---
In reply to: [86872966](https://github.com/Azure/azure-powershell/pull/3170#discussion_r86872966) [](ancestors = 86872966)